realsense/scene_perception/win/scene_perception_object.cc

```diff
@@ -493,50 +478,12 @@ void ScenePerceptionObject::OnRunPipeline() {
     state_ = IDLE;
     return;
   }
-
-  PXCCapture::Sample *sample = sense_manager_->QueryScenePerceptionSample();
-  if (!sample) {
-    // If the SP module is paused, the sample will be NULL.
-    // Query the raw color/depth images to support live preview and
-    // calculation of scene quality
-    sample = sense_manager_->QuerySample();
-  }
-
-  if (!sample || !sample->color || !sample->depth) {
-    triggerError("Failed to query sample.");
-
-    ReleaseResources();
-    state_ = IDLE;
+  PXCCapture::Sample* sample = QuerySample();
+  if (!sample)
     return;
```

Comment:
Sorry, I just notice that you are using <code>QuerySample()</code> not <code>sense_manager_->QuerySample()</code>. Please ignore the above comment.